src/index.js

```diff
@@ -88,6 +88,8 @@ export default class extends Component {
 
   onMessage = nativeEvent => this.props.onMessage({ nativeEvent });
 
+  injectedJavaScript = html => html && html.replace(/<body>/, `<body><script>${this.props.injectedJavaScript}</script>`);
```

Comment:
Please only do this if you have a prop `injectedJavaScript`.
Please rename the function to injectJavaScript.
Please inject the Javascript at the end of the body.

```js
injectJavaScript = html => this.props.injectedJavaScript && html ? html.replace('</body>', `<script>${this.props.injectedJavaScript}</script></body>`) : html;
```

Reply:
A new commit below `Fix injectedJavaScript() when this.props.injectedJavaScript is undefined` and inject the Javascript at the end of the body.

The reason of the name injectedJavaScript(html) but not injectJavaScript(js) is, injectJavaScript(js) ref to injectJavaScript(data) in [WebView.android.js](https://github.com/facebook/react-native/blob/master/Libraries/Components/WebView/WebView.android.js) is another featuer to be implemented here by someone :stuck_out_tongue:

Reply:
I see your point :) Maybe rename it to handleInjectedJavaScript?